var/spack/repos/builtin/packages/survey/package.py

```diff
@@ -68,6 +68,9 @@ class Survey(CMakePackage):
     depends_on("py-psutil", type=('build', 'run'))
     depends_on("py-sqlalchemy", type=('build', 'run'))
     depends_on("py-pyyaml", type=('build', 'run'))
+    depends_on("py-seaborn", type=('build', 'run'))
```

Comment:
FWIW, if only `survey@1.0.3:` needs these, you can add a `when` to this like

```
depends_on(py-seaborn, type=(...), when='@1.0.3:')
```

Do you think that would be useful?

Reply:
@scheibelp Yes, that is a good idea.   Thanks.  I will submit changes for that.